src/containers/BookList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+import { removeBook } from '../actions';
+
+const BookList = ({ books, dispatch }) => {
+  const handleRemoveBook = e => {
+    const { id } = e.currentTarget.dataset;
+    dispatch(removeBook(id));
```

Comment:
[OPTIONAL]: You can also use [`mapDispatchToProps`](https://react-redux.js.org/using-react-redux/connect-mapdispatch)